pkg/operator/ceph/nvmeof_recoverer/nvmeofstorage/controller.go

```diff
@@ -54,6 +54,21 @@ const (
 	FabricFailureDomainPrefix = "fabric-host"
 )
 
+// INITIALIZATION -> ACTIVATED
+type ControllerState int
+
+const (
+	INITIALIZATION = 0
```

Comment:
```suggestion
	INITIALIZATION = iota
```
go에서는 이렇게 많이 사용하는 것 같던데 혹시 어때보이시나요?

Reply:
Done